script/server

```diff
@@ -61,6 +61,34 @@ then
   fatal "config/diaspora.yml is missing! Copy over config/diaspora.yml.example to config/diaspora.yml and edit it properly!"
 fi
 
+# Check if git merge is in progress
+if [ -f .git/MERGE_MODE ]; then
+  fatal "A git merge is in progress!"
+fi
+
+# Check if detached head state
+git_branch_name="$(git symbolic-ref HEAD 2>/dev/null)"
```

Comment:
I think it's a good idea to skip this check if git is unavailable, like we do below.